packages/vscode-extension/src/dependency/DependencyManager.ts

```diff
@@ -8,6 +8,7 @@ import { getAppRootFolder } from "../utilities/extensionContext";
 import path from "path";
 import { getIosSourceDir } from "../builders/buildIOS";
 import { isExpoGoProject } from "../builders/expoGo";
+import { shouldUseExpoCLI } from "../../src/project/metro";
```

Comment:
Please move this function to utils and import in booth places I don't see the reason for it to live i metro files specifically